Assets/Scripts/InGameDisplay.cs

```diff
@@ -64,9 +66,16 @@ void OnGUI()
     //When it is clicked we need to disable the inventory canvas
     public void Back()
     {
-        backClicked = true;
+        inventoryCanvas.enabled = false;
```

Comment:
This class is outdated and was refactored for the state pattern. Your branch doesn't have my changes pulled in. The states are now doing the work.